src/main/scala/com/github/simplesteph/ksm/source/BitbucketServerSourceAcl.scala

```diff
@@ -68,7 +76,7 @@ class BitbucketServerSourceAcl extends SourceAcl {
     )
     request.header("Authorization", s"Basic $basicB64")
 
-    val response: Response = HTTP.get(request)
+    val response: Response = http.get(request)
```

Comment:
why refactoring the `HTTP` into `http`?

Reply:
i need override http in tests to mock get requests for Unit testing. I did not find how to mock properly HTTP static object.